static/sass/print.scss

```diff
@@ -91,7 +95,8 @@ footer,
 .global-nav__header-list,
 .dropdown-window,
 .dropdown-window-overlay,
-.p-navigation--secondary {
+.p-navigation--secondary,
+.p-table-of-contents {
```

Comment:
We can remove this whole sidebar so let's target `.p-sticky-toc` instead of `.p-table-of-contents`